bpfman-operator/hack/build-release-yamls.sh

```diff
@@ -33,54 +33,54 @@ KUSTOMIZE=${LOCALBIN}/kustomize
-    echo "#" >> release/bpfman-crds-install-v${VERSION}.yaml
-    echo "# $file" >> release/bpfman-crds-install-v${VERSION}.yaml
-    echo "#" >> release/bpfman-crds-install-v${VERSION}.yaml
-    cat $file >> release/bpfman-crds-install-v${VERSION}.yaml
+    echo "---" >> release-v${VERSION}/bpfman-crds-install.yaml
+    echo "#" >> release-v${VERSION}/bpfman-crds-install.yaml
+    echo "# $file" >> release-v${VERSION}/bpfman-crds-install.yaml
+    echo "#" >> release-v${VERSION}/bpfman-crds-install.yaml
+    cat $file >> release-v${VERSION}/bpfman-crds-install.yaml
 done
 
-echo "Generated:" release/bpfman-crds-install-v${VERSION}.yaml
+echo "Generated:" release-v${VERSION}/bpfman-crds-install.yaml
 
 ## 2. bpfman-operator install yaml
 
 $(cd ./config/bpfman-operator-deployment && ${KUSTOMIZE} edit set image quay.io/bpfman/bpfman-operator=quay.io/bpfman/bpfman-operator:v${VERSION})
-${KUSTOMIZE} build ./config/default > release/bpfman-operator-install-v${VERSION}.yaml
+${KUSTOMIZE} build ./config/default > release-v${VERSION}/bpfman-operator-install.yaml
 ### replace configmap :latest images with :v${VERSION}
-sed -i "s/quay.io\/bpfman\/bpfman-agent:latest/quay.io\/bpfman\/bpfman-agent:v${VERSION}/g" release/bpfman-operator-install-v${VERSION}.yaml
-sed -i "s/quay.io\/bpfman\/bpfman:latest/quay.io\/bpfman\/bpfman:v${VERSION}/g" release/bpfman-operator-install-v${VERSION}.yaml
+sed -i "s/quay.io\/bpfman\/bpfman-agent:latest/quay.io\/bpfman\/bpfman-agent:v${VERSION}/g" release-v${VERSION}/bpfman-operator-install.yaml
+sed -i "s/quay.io\/bpfman\/bpfman:latest/quay.io\/bpfman\/bpfman:v${VERSION}/g" release-v${VERSION}/bpfman-operator-install.yaml
 
-echo "Generated:" release/bpfman-operator-install-v${VERSION}.yaml
+echo "Generated:" release-v${VERSION}/bpfman-operator-install.yaml
 
 ## 3. examples install yamls
 
 ### XDP
-${KUSTOMIZE} build ../examples/config/v${VERSION}/go-xdp-counter > release/go-xdp-counter-install-v${VERSION}.yaml
-echo "Generated:" go-xdp-counter-install-v${VERSION}.yaml
+${KUSTOMIZE} build ../examples/config/v${VERSION}/go-xdp-counter > release-v${VERSION}/go-xdp-counter-install.yaml
+echo "Generated:" release-v${VERSION}/go-xdp-counter-install.yaml
 ### TC
-${KUSTOMIZE} build ../examples/config/v${VERSION}/go-tc-counter > release/go-tc-counter-install-v${VERSION}.yaml
-echo "Generated:" go-tc-counter-install-v${VERSION}.yaml
+${KUSTOMIZE} build ../examples/config/v${VERSION}/go-tc-counter > release-v${VERSION}/go-tc-counter-install.yaml
+echo "Generated:" release-v${VERSION}/go-tc-counter-install.yaml
 ### TRACEPOINT
-${KUSTOMIZE} build ../examples/config/v${VERSION}/go-tracepoint-counter > release/go-tracepoint-counter-install-v${VERSION}.yaml
-echo "Generated:" go-tracepoint-counter-install-v${VERSION}.yaml
+${KUSTOMIZE} build ../examples/config/v${VERSION}/go-tracepoint-counter > release-v${VERSION}/go-tracepoint-counter-install.yaml
+echo "Generated:" release-v${VERSION}/go-tracepoint-counter-install.yaml
```

Comment:
Need to add kprobe and uprobe, might need go-target.

Reply:
Good catch I think I got it!